### PR TITLE
Student Home Screen

### PR DESCRIPTION
**Pull request by s-oswald**
Updated student home screen layout, completed the "articles for you" portion, and added initial framework for remaining parts

### CODE REVIEW

File: lib/student/studentscreen.dart [hunk 1]
```diff
@@ -29,10 +29,10 @@ class _StudentScreenState extends State<StudentScreen> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(
-        title: Text("Student Screen"),
-        backgroundColor: Color.fromRGBO(75, 209, 160, 1),
-      ),
+      // appBar: AppBar(
```
Comment (chwerth) [inline lib/student/studentscreen.dart:32]: Do we not need this now? Should we just remove it?

File: lib/mentor/home.dart [hunk 2]
```diff
@@ -1,90 +1,90 @@
...
+                        padding: EdgeInsets.fromLTRB(10, 10, 10, 10),
+                      ),
+
+                      RaisedButton(
+                        child: Text(" List of Mentees "),
+                        onPressed: tmpFunction,
+                        color: Color(0xff096BD8),
+                        textColor: Colors.white,
+                        splashColor: Colors.grey,
+                        padding: EdgeInsets.fromLTRB(10, 10, 10, 10),
```
Comment (chwerth) [inline lib/mentor/home.dart:79]: These buttons seem mostly copy pasted, right? Can we pull them out into a function?

**Review by chwerth — CHANGES_REQUESTED** [review]
Looks good, great work! Only have two comments

File: lib/student/studentscreen.dart (hunk 1 above)
Comment (s-oswald) [reply lib/student/studentscreen.dart:32]: I left it as a placeholder while I was messing around with the layouts - I can remove it with next PR

File: lib/mentor/home.dart (hunk 2 above)
Comment (s-oswald) [reply lib/mentor/home.dart:79]: I didnt touch Mentor screen at all, just left as placeholders until Prep tells us what they wanted the mentor screen to look like

**Comment by chwerth** [comment]
~~Oh, also, would you remove the `.DS_Store` file? And add that to the .gitignore? I'm pretty sure that's a mac thing that we don't need to source control.~~

File: lib/student/studentscreen.dart (hunk 1 above)
Comment (chwerth) [reply lib/student/studentscreen.dart:32]: Ok.

File: lib/mentor/home.dart (hunk 2 above)
Comment (chwerth) [reply lib/mentor/home.dart:79]: Ok, sure. In that case I think just remove the .DS_Store file and add that to the .gitignore and we can go ahead and merge this

**Review by chwerth — APPROVED** [review]

**Review by ggpepp — APPROVED** [review]

**Pull request merged**